services/QuillLMS/app/controllers/api/v1/rule_feedback_histories_controller.rb

```diff
@@ -5,7 +5,7 @@ def by_conjunction
     raise ArgumentError unless params.include?('activity_id') && params.include?('conjunction')
 
     options = params.permit(:conjunction, :activity_id, :start_date, :end_date).to_h.symbolize_keys
-    report = RuleFeedbackHistory.generate_report(**options)
```

Comment:
Note: I will remove the old `RuleFeedbackHistory.generate_report` method (which used Postgres) after we're happy with the performance of this PR in production.